blog-post.html

```diff
@@ -4,9 +4,83 @@
+    </section>
+
+    <section id="main">
+        <div id="blog_post_img_container">
+          <img id="blog_post_img" src="/assets/mansfield.jpg"/>
+        </div>
+        <div class="blog_content_container">
+
+            <h3 class="totally_blog"><strong>Lorem ipsum dolor, sit amet!</strong><br><br>
+            </h3>
+          <p class="blog_content">
+              
+            Lorem, ipsum dolor sit amet consectetur adipisicing elit. Officia veniam sint impedit voluptates laborum tempora iure laudantium autem earum molestiae maiores distinctio pariatur officiis illum consequuntur quos, quis sapiente. Lorem ipsum dolor sit amet consectetur adipisicing elit. Quae id quo non ut magni placeat, optio natus maiores soluta ipsum tenetur ipsa, corrupti officiis, sed necessitatibus tempora voluptas dolorem adipisci! Lorem ipsum dolor sit amet consectetur adipisicing elit. Facere placeat eos magni inventore, impedit nemo harum quod architecto omnis aliquam accusantium corrupti minus, corporis, aperiam similique nisi necessitatibus debitis aut. Lorem ipsum dolor sit amet consectetur, adipisicing elit. Eos doloribus, corrupti iure nesciunt a autem error minima sunt doloremque rerum magnam? Dignissimos animi nam ullam sequi blanditiis libero nobis fugiat.<br><br>
+
+          
+          </p>
+                <h3 class="totally_blog"><strong>Lorem ipsum dolor, sit amet consectetur adipisicing.</strong><br><br>
+                </h3>
+          
+              <p class="blog_content">
+          
+                <p class="blog_content">
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Facere placeat eos magni inventore, impedit nemo harum quod architecto omnis aliquam accusantium corrupti minus, corporis, aperiam similique nisi necessitatibus debitis aut. Lorem ipsum dolor sit amet consectetur, adipisicing elit. Eos doloribus, corrupti iure nesciunt a autem error minima sunt doloremque rerum magnam? Dignissimos animi nam ullam sequi blanditiis libero nobis fugiat. Lorem ipsum, dolor sit amet consectetur adipisicing elit. Sapiente impedit repudiandae laudantium hic ab? Quisquam rerum, illum quo perspiciatis amet excepturi mol. Lorem, ipsum dolor sit amet consectetur adipisicing elit. Incidunt laborum in iusto, rem quo delectus ipsum reiciendis eum tenetur expedita distinctio quia assumenda nisi! Optio nesciu.<br><br>
+        
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptas, at! Velit enim, dolorem minus, cumque dignissimos exercitationem non quaerat iusto inventore omnis qui consequuntur autem voluptatibus dicta libero ipsum sit? Lorem ipsum dolor sit amet, consectetur adipisicing elit. Aliquid natus quod in exercitationem, aspernatur eos nihil nulla voluptatem, voluptate asperiores omnis consectetur culpa consequuntur rerum at aut possimus magnam dignissimos?
+          </p>
+        </div>
+    </section>
+
+    <section id="footer">

```

Comment:
This is a great footer, the code is easy to understand and the section names are very clear about what it is that they hold. Good job